From 0e92519878ae1721db1f3bd03ea076b94b0e267e Mon Sep 17 00:00:00 2001
From: manny <manny@legworkprm.com>
Date: Wed, 25 May 2022 13:57:31 -0700
Subject: [PATCH 1/2] add default for additionalProperties if not provided

---
 middleware/swagger-validator.js | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/middleware/swagger-validator.js b/middleware/swagger-validator.js
index c968cdc..bd43bc2 100644
--- a/middleware/swagger-validator.js
+++ b/middleware/swagger-validator.js
@@ -140,7 +140,17 @@ var validateValue = function (req, schema, path, val, location, callback) {
                                                     schema.type), aVal, oCallback);
       } else {
         try {
-          validators.validateAgainstSchema(schema.schema ? schema.schema : schema, val);
+          const schemaToValidate = schema.schema ? schema.schema : schema;
+          const additionalProperties = schemaToValidate.additionalProperties;
+
+          // if additionalProperties has been left off of the schema definition
+          // then we want to enforce a default of false (versus the unforked repo's default of true),
+          // preventing additional attributes from being passed into the POST/PATCH request body
+          // and accepted as valid by default.
+          if (additionalProperties === undefined) {
+            schemaToValidate.additionalProperties = false;
+          }
+          validators.validateAgainstSchema(schemaToValidate, val);
 
           oCallback();
         } catch (err) {
-- 
2.25.1

